lib/inspec/reporters/base.rb

```diff
@@ -0,0 +1,23 @@
+module Inspec::Reporters
+  class Base
+    attr_reader :run_data
+
+    def initialize(config)
+      @run_data = config[:run_data]
+      @output = ''
+    end
+
+    def output(str)
+      @output << "#{str}\n"
```

Comment:
Alternatively, one could make `@output` an array that we continually `<<` onto, and `rendered_output` can `join("\n")`

Reply:
This is a good note. I think I prefer to keep it a string at this time but will keep this idea for the future.